auth.go

```diff
@@ -1,15 +1,13 @@
 package winrm
 
 import (
+	"crypto/tls"
 	"fmt"
-	"io/ioutil"
 	"net"
+	"net/http"
 	"strings"
 	"time"
 
-	"github.com/masterzen/azure-sdk-for-go/core/http"
```

Comment:
Removing this means we're breaking the compatibility with Go < 1.8. 
But maybe we don't care anymore nowadays...